### PR TITLE
Topology-Edit Workload and Resource Type

### PR DESCRIPTION
**Pull request by gajanan-more**
**Fix:** https://issues.redhat.com/browse/ODC-5583 

**Problem:** Automation to edit workload and edit resource types of workload

**Solution:** This script will delete the already created workload using the Actions dropdown menu and Context Menu

**Cluster Setup:**
```

- export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
- BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
- BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.dev-svc-4.8-042801.devcluster.openshift.com
- export BRIDGE_KUBEADMIN_PASSWORD
- export BRIDGE_BASE_ADDRESS
- oc login -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
- oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
- oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
- ./test-cypress.sh -p topology

```

**Test Setup:**
1. Replace `"TAGS": "@smoke and not @manual and not @to-do"`, with ` "@Topology and (@smoke or @regression) and not (@Manual or @to-do)",` in `topology/cypress.json` under `env`
2. Run `yarn run test-cypress-topology` in frontend
3. Execute feature file: `frontend/packages/topology/integration-tests/features/topology/topology-editing-app-node.feature`

**Test Execution Screenshot:**

Screenshot from 2021-05-21 12-42-57

### CODE REVIEW

**Review by makambalaji — CHANGES_REQUESTED** [review]
Execute 'Yarn run lint' on topology/integration-tests folder

File: frontend/packages/dev-console/integration-tests/support/pages/app.ts [hunk 1]
```diff
@@ -44,6 +44,7 @@ export const perspective = {
 export const navigateTo = (opt: devNavigationMenu) => {
   switch (opt) {
     case devNavigationMenu.Add: {
+      perspective.switchTo(switchPerspective.Developer);
```
[the remark this answers is not in the document]
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/support/pages/app.ts:47]: Sometimes we left at some other pages and to ensure that we are on the Add page, I have added that line.

File: frontend/packages/dev-console/integration-tests/support/pages/operators-page.ts [hunk 2]
```diff
@@ -67,6 +67,14 @@ export const operatorsPage = {
     }).should('be.visible');
   },
 
+  verifyNoOperatorsInstalled: (): boolean => {
+    if (cy.get(operatorsPO.installOperators.noOperatorFoundMessage)) {
+      return true;
+    }
+
+    return false;
+  },
```
[the remark this answers is not in the document]
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/support/pages/operators-page.ts:76]: Actually, this is not needed. I was trying something else. Removing it.

**Comment by gajanan-more** [comment]
> Execute 'Yarn run lint' on topology/integration-tests folder
I don't find anything post running it on topology/integration-tests folder
> 
> Update the TAGS as "@Topology and (@smoke or @regression) and not (@Manual or @to-do)",
Updated.

File: frontend/packages/dev-console/integration-tests/support/pages/app.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/support/pages/app.ts:47]: Lets add it to the step definition, please don't change the functionality to

File: frontend/packages/dev-console/integration-tests/support/pages/operators-page.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/support/pages/operators-page.ts]: ok

**Comment by makambalaji** [comment]
/lgtm
/approve
image

**Pull request merged**